src/app.js

```diff
@@ -48,6 +51,16 @@ const httpApp = createGatewayApp(httpServer);
 let httpsServer = createHttpsServer();
 let httpsApp = null;
 
+
+/**
+ * Creates an HTTPS server object, if successful. If there are no public and
+ * private keys stored for the tunnel service, null is returned.
+ *
+ *
+ * @param {}
+ * @return {Object|null} https server object if successful, else NULL}
```

Comment:
nit: remove trailing `}`

Reply:
done. I’ve also updated other comments to suit this format.